### DISCUSSION TITLE
How to add devices via router.

### DISCUSSION THREAD

**Discussion by ChengyiP** [Koenkk/zigbee2mqtt#8069] [Devices]
Hi All,

I am new to home automation stuff. I would like to know if there is any way that i can add devices via a router. Currently i m using CC2531 as a coordinator, and another cc2531 as a router, another ikea repeater in my network. Also on top of that, i have a bunch of other end devices, as well as plugged-in devices(router). E.g Sonoff's BASICZBR3, ZBMINI, Tuya's LED controller.  Currently, i have a total of 45 devices. And i m not able to add any more devices to my network. 

I know that there is a limit as to how many devices that can be connected directly to the coordinator, and it appears to me that especially the aqara sensors love the coordinator, despite having the ikea repeater or the cc2531 router in close proximity.

In short, would like to know if it is possible to add devices via (e.g) SONOFF's BASICZBR3. If yes, how can it be done. 

Thanks in advance.

**Comment by iotinkognito** [comment]
Also having a similar problem. I have CC2531 coordinator and two IKEA TRADFRI signal repeaters E1746. I mostly use aqara sensors. The ikea repeaters always show LQI of 0 in zigbee2mqtt even when less than 5meters line of sight to the coordinator. Aqara sensors always only connect to coordinator directly and some of them are ~20 meters away around a few corners.

**Reply by ChengyiP** [reply]
Hey, just take note on the total amount of devices on your network, i kinda crashed my network by overloading it. I have switch to ZHA for now with a sonoff flashed tasmota bridge. Still loving Z2M, much more stable in my opinion. Waiting for my new cc2652 stick to arrive before moving it back to z2m

**Reply by iotinkognito** [reply]
I have 2x slaeh stick and the LQI is even lower

**Reply by ChengyiP** [reply]
I wouldnt worry too much about the LQI.. If it is all working well. Just dont look into the map. I spend too much time looking at the map which i have zero control of it. If it is not giving u problem, then just leave it. I remember seeing it somewhere that koenkk did mentioned that these number may not be a 100% representation of the actual figure. So, yep... i m enjoying my setup now.
